src/styles/blocks/page.scss

```diff
@@ -0,0 +1,4 @@
+.page__body {
+  margin: 0;
+  font-family: Roboto, sans-serif, system-ui;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the font-family should not include generic family names if they are not used. In this case, 'system-ui' is a generic family name and should be removed if 'Roboto' and 'sans-serif' cover the font styles needed for the project.